source-browser.html

```diff
@@ -62,6 +87,44 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
     <a class="fixed right-0 bottom-0 mr2 mb2 text-decoration-none h2" id="refresh" href="#">🔄</a>
 
     <script type="text/javascript" src="https://unpkg.com/zepto@1.2.0/dist/zepto.js" integrity="sha384-Cp3V2nlfJJ5aA0ctd1PkfNAEMkXM0EGa6RrmEgiv8D6TCaeZJfUFs/PYfR+B5F5H" crossorigin="anonymous"></script>
+    <script type="text/javascript">
+      setInterval(function(){ 
+          $('#content').height($(window).height() - $('#navbar').height() - $('.stickTopic').height() - 40);
+          $('.repos').each(function(index) {
+              var h = $(window).height() - $('.stickTopic').height() - 8;
```

Comment:
Don't use `var` Prettier will also fix this.

Reply:
it didn't!!

Reply:
Prettier didnt fixed it, but i do, so done! :)